### ISSUE TITLE
Allow notifications email address to be configurable

### ISSUE THREAD

**Issue by jgutix** [loomio/loomio#5539]
Notifications email being hardcoded here: https://github.com/loomio/loomio/blob/ae1f4affe916cfaabff156ecc34fff11fe172f5f/app/mailers/base_mailer.rb#L18 is conflicting with Rackspace SMTP server as they [require the "from" field being the same as the username](https://stackoverflow.com/questions/22699768/actionmailer-netreadtimeout-when-using-delayed-job-with-rackspace-emailsrvr), so for example this configuration:
```
# smtp settings
SUPPORT_EMAIL=loomio@myserver.org

SMTP_AUTH=login
SMTP_DOMAIN=myserver.org
SMTP_SERVER=secure.emailsrvr.com
SMTP_PORT=465
SMTP_USERNAME=loomio@myserver.org
SMTP_PASSWORD=******
SMTP_USE_SSL=true

REPLY_HOSTNAME=myserver.org

# helper bot is the account which welcomes people to their groups.
HELPER_BOT_EMAIL=loomio@myserver.org
```
will result in an error like this
```
[Worker(host:9d0144b9a7c5 pid:8)] Job UserMailer.login (id=25) FAILED (0 prior attempts) with Net::SMTPFatalError: 550 5.1.0 <notifications@mydomain.org>: Sender address rejected: User unknown in relay recipient table
loomio-worker  | 2019-06-10T05:19:49+0000: [Worker(host:9d0144b9a7c5 pid:8)] Job UserMailer.login (id=25) FAILED (0 prior attempts) with Net::SMTPFatalError: 550 5.1.0 <notifications@mydomain.org>: Sender address rejected: User unknown in relay recipient table
```

I tried config entries like: 
`NOTIFICATIONS_EMAIL=loomio@myserver.org`
`NOTIFICATIONS_EMAIL_ADDRESS=loomio@myserver.org`
And didn't seem to work

In short my instance isn't working because I can't receive the code that's supposed to be sent via email. Ultimately this is my first time trying loomio, so if there's any other thing I can try, please let me know.

**Comment by robguthrie** [comment]
Hi @jgutix thanks for trying Loomio and reporting this issue.

I've made a PR to address this, when it's in master I'll ping you. Be sure to change from "stable" to latest in docker-compose app and worker sections.

**Comment by robguthrie** [comment]
@jgutix you can now set NOTIFICATIONS_EMAIL_ADDRESS in your env!
